euler/run-demand.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env sh
+
+# parameters
```

Comment:
actually, the script is super-straightforward. I don't believe adding inline comments would make it easier. In fact, inline comments can also be harmful, as they are often not updated when changes to the code are made (you know that, you have done this too, as has everyone, that is why naming your variables and functions properly is sooo much more important!) I can make a header comment at the top of the file with a quick description of what is going on, I guess...